source/iostreams.tex

```diff
@@ -10036,8 +10036,8 @@
 A sequence of elements that identify
 the location of a file within a filesystem.
 The elements are the
-\grammarterm{root-name\opt},
-\grammarterm{root-directory\opt},
+\grammarterm{\opt{root-name}},
+\grammarterm{\opt{root-directory}},
```

Comment:
@burblebee: It's an express change in this patch series to switch from `blah\opt{}` to `\opt{blah}`, because it makes it less likely to get a spacing accident by forgetting the empty `{}`.